app/main/views/suppliers.py

```diff
@@ -471,3 +481,80 @@ def update_supplier_framework_details(supplier_id, framework_slug):
         return jsonify(message="Database Error: {0}".format(e)), 400
 
     return jsonify(frameworkInterest=interest_record.serialize()), 200
+
+
+@main.route('/suppliers/<int:supplier_id>/frameworks/<framework_slug>/variation/<variation_slug>', methods=['POST'])
```

Comment:
I was also thinking that if we were being really rest-y it would be good to accept `GET` to this endpoint which would just return the response (or... 404? if not agreed) bit it wouldn't really add anything that you don't get from the supplierframework endpoint anyway.